tsdb/head_test.go

```diff
@@ -644,9 +644,9 @@ func TestDeleteUntilCurMax(t *testing.T) {
 	testutil.Assert(t, res.Next(), "series don't exist")
 	exps := res.At()
 	it := exps.Iterator()
-	ressmpls, err := expandSeriesIterator(it)
+	ressamples, err := expandSeriesIterator(it)
```

Comment:
If you're going to unabbreviate `ressmpls` here, I'd probably go for `resSamples` camelCasing.